src/SIL.XForge/Services/AuthService.cs

```diff
@@ -40,6 +40,19 @@ public bool ValidateWebhookCredentials(string username, string password)
 
     public Task<string> GetUserAsync(string authId) => CallApiAsync(HttpMethod.Get, $"users/{authId}");
 
+    public Task<string> GenerateTransparentUser(string name, string username, string password, string language)
```

Comment:
How about anonymous user? The authentication is transparent, the user is anonymous.